src/frontend/components/Header/Header.js

```diff
@@ -1,19 +1,24 @@
 import React from "react";
-import logo from "./image/logo/WePair7.svg";
-import user from "./image/userIcon.svg";
-import "./App.css";
+import logo from "../../image/logo/WePair7.svg";
+import user from "../../image/userIcon.svg";
+import "./Header.css";
 
 export default function Header() {
   return (
     <div className="Header">
-      <div className="Header-top">
+      <div className="navbar">
         <div className="logo">
           <img className="logo-img" src={logo} alt="logo" />
         </div>
-        <div className="signing">
-          <a className="sign-up">Sign Up </a>
+        <div className="navbar-div">
+          <span className="sign-up" >
+            Sign Up{" "}
```

Comment:
why do we need `{" "}` in the text ?

Reply:
That's a space in between sign up | long in 
maybe when we do the UI we can change it!